Dockerfile-default-rocm

```diff
@@ -63,13 +148,63 @@ ARG HOROVOD_WITH_TENSORFLOW=1
 ARG HOROVOD_WITH_PYTORCH=1
 ARG HOROVOD_WITHOUT_MXNET=1
 ARG HOROVOD_GPU_OPERATIONS=NCCL
-ARG HOROVOD_WITHOUT_MPI=1
+ARG HOROVOD_WITHOUT_MPI=0
+ARG HOROVOD_WITH_MPI=1
 ARG HOROVOD_GPU=ROCM
-ARG HOROVOD_WITHOUT_MPI=1
-ENV LD_LIBRARY_PATH=/opt/rocm/lib:/opt/rocm/hip/lib
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH;/opt/rocm/lib:/opt/rocm/hip/lib
+
+ENV HOROVOD_PIP $HOROVOD_PIP
+ENV HOROVOD_WITH_TENSORFLOW $HOROVOD_WITH_TENSORFLOW
+ENV HOROVOD_WITH_PYTORCH $HOROVOD_WITH_PYTORCH
+ENV HOROVOD_WITHOUT_MXNET $HOROVOD_WITHOUT_MXNET
+ENV HOROVOD_GPU_OPERATIONS $HOROVOD_GPU_OPERATIONS
+ENV HOROVOD_WITHOUT_MPI $HOROVOD_WITHOUT_MPI
+ENV HOROVOD_WITH_MPI $HOROVOD_WITH_MPI
+ENV HOROVOD_GPU $HOROVOD_GPU
+ENV HOROVOD_NCCL_HOME $HOROVOD_NCCL_HOME
+ENV NCCL_LIB_DIR=${HOROVOD_NCCL_HOME}/lib
+ENV HOROVOD_NCCL_LINK=${WITH_OFI:+SHARED}
+ENV LD_LIBRARY_PATH=${WITH_OFI:+$NCCL_LIB_DIR:}$LD_LIBRARY_PATH
+
 RUN if [ "$HOROVOD_PIP" != "0" ]; then pip install "${HOROVOD_PIP}" ; fi
 
-RUN rm -r /tmp/*
+RUN pip install -r /tmp/det_dockerfile_scripts/additional-requirements.txt
+
+RUN pip uninstall -y tb-nightly tensorboardX
+RUN pip install tensorboard==2.10.1
+
+RUN pip install pynvml 
```

Comment:
see `additional-requirements.txt`

Reply:
moved to `additional-requirements-rocm.txt`